react-delegationdashboard/src/config.devnet.ts

```diff
@@ -27,6 +27,7 @@ export const stakingContract: string =
   'erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqllls0lczs7';
 export const delegationManagerContract: string =
   'erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqylllslmq6y6';
+export const walletConnectBridge: string = 'https://walletconnect-bridge.maiarbrowser.com';
```

Comment:
We shuold not include this address. as default bridge address. This is for us as an internal test bridge. There is a generic walletconnect bridge address, or a http://localhost:5001 (this is the default when starting locally the bridge)

Reply:
Done